examples/demos/utils.go

```
@@ -44,33 +52,46 @@ func Step(msg string) {
 	fmt.Println()
 	fmt.Printf("%s%s%s\n", blueBold, msg, reset)
 	fmt.Printf("Press enter to continue...")
-	fmt.Scanln()
+	if _, err := fmt.Scanln(); err != nil {
+		log.Fatal(err)
```

We could probably safely ignore this error and just `log.Print()` it

---

Done